src/services/reciters/custom.adapter.ts

```
@@ -0,0 +1,55 @@
+import { getRiwayaKeyFromMoshafName } from './mp3quran.helpers';
+import type { ReciterSource } from './reciter-source';
+import { retryFetch } from './shared-fetch';
+import type { Playlist, Reciter } from '../../types';
+import type { CustomApiReciter, CustomApiResponse } from './custom.types';
```

ESLint

[error] `./custom.types` type import should occur before import of `./mp3quran.helpers`

(import/order)